src/Workspaces/Core/Portable/Shared/Extensions/ICodeDefinitionFactoryExtensions.cs

```diff
@@ -350,7 +350,40 @@ private static bool TryGetValue(IDictionary<string, ISymbol> dictionary, string
                 name: overriddenEvent.Name);
         }
 
-        public static async Task<IMethodSymbol> OverrideMethodAsync(
+        public static async Task<ISymbol> OverrideAsync(
+            this SyntaxGenerator generator,
+            ISymbol symbol,
+            INamedTypeSymbol containingType,
+            Document document,
+            DeclarationModifiers? modifiersOpt = null,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var modifiers = modifiersOpt ?? symbol.GetSymbolModifiers();
+            modifiers = modifiers.WithIsOverride(true)
```

Comment:
Shouldn't the underlying generator functions be doing this? Why would I call GenerateMethodAsync() and _not_ do this?